packages/server/src/threads/automation.ts

```diff
@@ -39,9 +43,6 @@ import env from "../environment"
 import tracer from "dd-trace"
 
 threadUtils.threadSetup()
-const FILTER_STEP_ID = actions.BUILTIN_ACTION_DEFINITIONS.FILTER.stepId
```

Comment:
These references to `BUILTIN_ACTION_DEFINITIONS` ended up causing an import cycle at one point so I removed them from the top level of the file.